upload.py

```diff
@@ -1111,11 +1111,24 @@ def handle_asset_upload(task: tasks.Task):
 def handle_asset_metadata_upload(task: tasks.Task):
     asset = eval(f"{task.data['export_data']['eval_path']}.blenderkit")
 
-    if task.status == 'finished':
-        asset.asset_base_id = task.data['asset_data']['assetBaseId']
-        asset.id = task.data['asset_data']['id']
+    if task.status != 'finished':
+        return
+
+    new_asset_base_id = task.data['asset_data'].get('assetBaseId')
```

Comment:
@vilemduha Question here is - can server assign new assetBaseId or ID?

`task.data['asset_data'].get('assetBaseId')` is assigned on first upload
`asset.asset_base_id = task.data['export_data']['assetBaseId']` is assigned on reuploads of downloaded asset